qiskit/transpiler/passmanager_config.py

```diff
@@ -42,7 +42,6 @@ def __init__(
         hls_config=None,
         init_method=None,
         optimization_method=None,
-        optimization_level=None,
```

Comment:
Was this just left in by accident in #8305? https://github.com/Qiskit/qiskit-terra/pull/8305/files#r954804111 Agree with removing, just want to make sure nothing depends on it.

Reply:
Yeah it was an oversight in #8305 nothing is using it anywhere it was only added by mistake because when we refactored away from using `optimization_level` from the passmanager_config I forgot to delete this. I only caught it here because in earlier iterations of this PR I added a `post_translation_pass_manager` kwarg to support passing the passmanager from the backend to the preset passmanager construction (which was removed as part of moving to plugins).